Assets/MixedRealityToolkit/_Core/Interfaces/Diagnostics/IMixedRealityDiagnosticsManager.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Core.Definitions.Diagnostics;
+using Microsoft.MixedReality.Toolkit.Core.Interfaces.Events;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Core.Interfaces.Diagnostics
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public interface IMixedRealityDiagnosticsManager : IMixedRealityEventSystem, IMixedRealityEventSource
+    {
+        /// <summary>
+        /// Enable / disable the diagnostic display
+        /// </summary>
+        bool Visible { get; set; }
```

Comment:
We prob don't want people setting these values, let's remove all the setters for the interface.

Reply:
It fits the pay-to-play mentality. Diagnostics aren't free (although likely not that expensive), so if you only want to see CPU usage or FPS it would be best to only allow showing that. I'd prefer to keep the knobs open if possible

Reply:
Agreed, but we do keep the knobs open by allowing devs to write their own implementation using the interface. We don't want another system outside of the diagnostics chaining values on us.

Reply:
I think I misunderstood the data binding between UI and manager implementations. Will set to readonly. Agree we don't want random systems messing with this